ftps-connector/modules/ROOT/nav.adoc

```diff
@@ -1,2 +1,2 @@
 * xref:ftps-connector.adoc[FTPS Connector User Guide]
-
+** xref:use-case-example.adoc[Use Case Example - Hello World]
```

Comment:
I think I'd go with a structure here similar to the one we're talking about for JDE, something like this:

*  FTPS Connector User Guide
** Intro
** Prerequisites
** Installation
** Configuration
** Use Cases
** Use Case Example

I think the User Guide page is short enough that you can add the Use Case Example page to it (as an H2, bumping all the headings it contains down a level), but we an talk about that.